pkg/executors/docker_compose_executor_test.go

```diff
@@ -112,6 +116,11 @@ func Test__DockerComposeExecutor(t *testing.T) {
 		"etc exists, multiline huzzahh!\n",
 		"Exit Code: 0",
 
+		"directive: " + kvmCmd,
+		"kvm\n",
+		"Exit Code: 0",
+
+
 		"directive: Exporting environment variables",
```

Comment:
These are unit test for testing if the compose class is executing the commands in the proper order. For testing out if KVM is introduced I advise introducing a dedicated test in the `test/e2e` directory.

Reply:
E2e in place, namely check_dev_kvm.